### ISSUE TITLE
I don't feel part of pull request #9399 is a bug

### ISSUE THREAD

**Issue by 43897689431805741092** [endless-sky/endless-sky#9996]
### Problem Description

See: https://github.com/endless-sky/endless-sky/pull/9939#issuecomment-2054196435

Sorry if making an Issue of this is not appropriate but since the PR has been merged and realistically very few people are going to read the comments after that fact(?), this seems like what I am supposed to do(?).

### Related Issue Links

https://github.com/endless-sky/endless-sky/pull/9939

### Desired Solution

Reversion of part of the PR's changes:
remove outfitter "Lovelace Basics"
remove outfitter "Syndicate Basics"
add outfitter "Delta V Basics"
add outfitter "Kraz Basics"

### Alternative Approaches

Bene-dictator did say "I did mention making a small, regional outfitter for the area around New China, though it wasn't picked up on by the developers. I do feel like there could be a way to tie it to the civil war – maybe the regional outfitters being selling sensor equipment in response to growing tensions and merchants' desires to protect themselves from piracy."

I don't think that would make me happy without more extensive changes that may be beyond the scope of this Issue that actually do "tie it to the civil war", but that's worth considering as it doesn't appear to have been.

**Comment by bene-dictator** [comment]
I don't think it makes much sense that Delta V and Kraz basics are sold so far away from their home regions, especially when it doesn't occur elsewhere.

How extensive would you like the changes to be, in regards to the new outfitter idea? I was thinking of at least a conversation talking about it. Conversations relating to the consequences of the nuclear bombings are incredibly scarce. I'm happy to write something up and PR it, I only said that "it wasn't picked up on by the developers" before because of Amazinite's reaction in the original PR.

The bottom line for me is that, if a differentiation in outfit stocks can be explained somewhere (e.g. a conversation), that would not only justify the inclusion but add some more life to the game where there was none before. Whereas before it appeared to be a mistake.

**Comment by bene-dictator** [comment]
Perhaps something like this:

```
mission "Central Outfitters Expansion Trigger"
	landing
	invisible
	to offer
		has "event: initial deployment 2"
	on offer
		event "central outfitters expansion" 45

event "central outfitters expansion"
	planet "Earth"
		add outfitter "Central Outfits Expansion"
	planet "Luna"
		add outfitter "Central Outfits Expansion"
	planet "New China"
		add outfitter "Central Outfits Expansion"
	planet "New Wales"
		add outfitter "Central Outfits Expansion"
	planet "Heartland"
		add outfitter "Central Outfits Expansion"

mission "Central Outfitters Expansion"
	outfitter
	source
		attributes "near earth" "dirt belt"
		outfits "Tactical Scanner"
		outfits "Caldera Afterburner"
		outfits "S3 Thermionic"
		outfits "Heavy Rocket"
		outfits "Meteor Missile"
		outfits "Javelin"
		outfits "Javelin Pod"
		outfits "Javelin Mini Pod"
		outfits "Javelin Storage Crate"
		outfits "Small Radar Jammer"
	to offer
		has "event: central outfitters expansion"
		not "event: start of hostilities"
	on offer
		conversation
			`When you walk into <origin>'s spaceport outfitter you notice it has new outfits for sale. To your surprise, almost all the new stock appears to be Delta V branded - a company you'd usually see selling in the Southern Rim, not far up here near the center of the Republic. It is especially strange as Delta V is one of the major companies supporting the Free Worlds secessionists.`
			choice
				`	(Ask the proprietor about the new outfits.)`
				`	(Move on.)`
					decline
			`	The proprietor's busy at their desk, but as you approach they look up and acknowledge you with "Hi, how can I help you today?"`
			choice
				`	"How come you're selling Delta V outfits up here?"`
				`	"It looks like you've got new stock."`
					goto "new stock"
				`	(Leave the proprietor alone.)`
					decline
[...]
```
Still working on it, but that's a potential PR.

**Comment by bene-dictator** [comment]
@43897689431805741092 Do you have any thoughts now that #10021 has been merged?

**Comment by bene-dictator** [comment]
See: https://discord.com/channels/251118043411775489/266345072554016768/1236518662794383411

Closed by #10021.

**Comment by 43897689431805741092** [comment]
> @43897689431805741092 Do you have any thoughts now that #10021 has been merged?

You asked me elsewhere and I'm happy with what you'd done! Just saying for the public record of this game's open-source development.

One thought I had but didn't share at the time was that I'd like to see the Greyhound and Impala Plasma engines still available, or at least the Greyhound ones since these also appear in the Core region under the Syndicate Advanced sales. The single most common case for me outfitting on New China has been "I want plasma engines on my freighters because they aren't using their battery reserves to power anything else 80% of the time", and that seems like a sentiment many merchants in-universe might share – the Bulk Freighter is noted as being painfully slow to align itself for hyperjumps, so plasma steering engines would be a perfect aftermarket refit for it.
